interface/patient_file/encounter/select_codes.php

```diff
@@ -82,9 +83,28 @@
                         "sLast": <?php echo xlj('Last'); ?>
                     }
                 },
+                fnInfoCallback: function(settings, start, end, max, total, pre) {
+                    const self = this.api();
+                    let json = self.ajax ? self.ajax.json() : null;
+                    if (json && json.iSearchEmptyError) {
+                        return <?php echo xlj('Please enter a search term to search for a code'); ?>;
+                    } else {
+                        return pre;
+                    }
+                },
+                fnDrawCallback: function(obj) {
+                    const infoBox = $(".dataTables_wrapper p.moreResults");
+                    if (infoBox.length) {
+                        infoBox.parent().remove();
+                    }
+                    if (obj.json && obj.json.iTotalHasMoreRecords) {
+                        $(".dataTables_wrapper .dataTables_info").parent().parent()
+                            .prepend("<div class='col-sm-12'><p class='alert alert-warning moreResults'>" + <?php echo xlj("Maximum displayable results reached.  Narrow your search"); ?> + "</p></div>");
```

Comment:
Avoid double spacing in translation constants.

Reply:
Fixed.